classes/class-wc-connect-paypal-ec.php

```diff
@@ -20,6 +20,11 @@ public function __construct( WC_Connect_Nux $nux ) {
 		public function init() {
 			if ( function_exists( 'wc_gateway_ppec' ) ) {
 				add_action( 'wp_loaded', array( $this, 'setup' ) );
+
+				if ( isset( $_GET['api_username'] ) ) {
```

Comment:
Is the intention here to schedule this order updating event when the API username is saved?

Does the `api_username` come in `GET` from the OAuth connection redirect?

Reply:
Yes https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/blob/master/includes/class-wc-gateway-ppec-ips-handler.php#L163 and I also intended this to happen when saving `api_username` manually, though I haven't tested that yet.